test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -278,6 +282,12 @@ public void setupSchema()
         createSchema();
     }
 
+    protected void cleanupRepairTables()
+    {
+        for (String table : Arrays.asList(SystemKeyspace.REPAIRS))
```

Comment:
IR keeps trying to load the system table, which can slow down the tests... so added this as a work around... it would be great if storage could be isolated and wasn't global state =(